ecs-agent/gmsacredclient/credentialsfetcherclient.go

```diff
@@ -18,6 +17,12 @@ type CredentialsFetcherClient struct {
 	timeout time.Duration
 }
 
+const (
+	// credentialsFetcherSocketLocation is the path to unix domain socket for
+	// communicating with credentials-fetcher daemon.
+	credentialsFetcherSocketLocation = "/var/credentials-fetcher/socket/credentials_fetcher.sock"
```

Comment:
Would this be an issue on Windows?

Reply:
credentialsfetcher is only linux, there won't be any issue with windows